sdk/packages/selector-base/src/types.ts

```diff
@@ -5,9 +5,11 @@ import { type Wallet } from '@wallet-standard/core'
 export interface Adapter {
   connect: () => Promise<void>
 }
+// TODO we have two types of QueryNetwork
 export enum QueryNetwork {
   SOLANA = 'SOLANA',
-  SUI = 'SUI'
+  SUI = 'SUI',
+  POLKADOT = 'POLKADOT'
```

Comment:
I wonder if parachains like aleph zero should have their own values here or if we will get info about which one should be used from session metadata